src/prospectScraper/info/2020ScheduledFallGames.csv

```diff
@@ -0,0 +1,241 @@
+School,Conference,2020ScheduledFallGames
+Air Force,American,1.0
```

Comment:
2020ScheduledFallGames should be an int value, not a float or double.
As previously stated, Air Force has 8 games scheduled.
```suggestion
Air Force,American,8
```

Reply:
Oh ok
This record refers to the game of 3/10/2020 in the American Athletic Conference, and if it has 8, they are classified by conferences,

 

>    Air Force, American, 1.0
>     Air Force, MWC, 6.0
>     Air Force, Ind, 1.0
 (6 + 1 + 1 = 8)

even so I already adjust the file to meet these new specifications :+1: